polaris_wyre/wyre/wyre.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple
+
+from polaris_wyre.wyre.dtos import TransferData
+from .api import TEST_BASE_URL, WyreAPI
+
+
+# TODO: add methods description
+class Wyre:
+    def __init__(
+        self, api_token: str = "", account_id: str = "", api_url: str = TEST_BASE_URL
+    ):
+        self.wyre_api = WyreAPI(
+            api_token=api_token, account_id=account_id, api_url=api_url
+        )
+
+    def get_account(self) -> Tuple[str, str]:
+        response_data = self.wyre_api.get_account()
+        return response_data["depositAddresses"]["XLM"].split(":")
+
+    def get_stellar_transaction_id(self, transfer_id) -> str:
+        transfer_is_completed = False
+        while not transfer_is_completed:
+            response_data = self.wyre_api.get_transfer_by_id(transfer_id)
+            if response_data["status"] == "FAILED":
+                # TODO: improve RuntimeError message with a better description
+                raise RuntimeError("Wyre failed to complete the transfer.")
+            if response_data["status"] == "COMPLETED":
```

Comment:
What about setting these `COMPLETED` and `FAILED` strings as constants?

Reply:
Sounds good!